app/src/main/java/org/covidwatch/android/data/contactevent/firebase/FirebaseContactEventFetcher.kt

```diff
@@ -25,11 +28,11 @@ class FirebaseContactEventFetcher(
 
     companion object {
         private const val TAG = "ResultFetcher"
+        private lateinit var listenerRegistration: ListenerRegistration
```

Comment:
let's move it out of `companion object`, it doesn't belong there. And we might need to make it nullable because we're not sure that it will be initialized:
```kotlin
companion object {}

private var listenerRegistration: ListenerRegistration? = null
```